discovery-provider/scripts/start.sh

```diff
@@ -7,22 +7,42 @@ if [[ -z "$audius_loggly_disable" ]]; then
         audius_discprov_hostname=$(echo $audius_discprov_url | sed -e 's/[^/]*\/\/\([^@]*@\)\?\([^:/]*\).*/\2/')
         audius_loggly_tags=$(echo $audius_loggly_tags | python3 -c "print(' '.join(f'tag=\\\\\"{i}\\\\\"' for i in input().split(',')))")
 
+        mkdir -p /var/log
         mkdir -p /var/spool/rsyslog
         mkdir -p /etc/rsyslog.d
-        cat >/etc/rsyslog.d/22-loggly.conf <<EOF
-\$WorkDirectory /var/spool/rsyslog # where to place spool files
-\$ActionQueueFileName fwdRule1   # unique name prefix for spool files
-\$ActionQueueMaxDiskSpace 1g    # 1gb space limit (use as much as possible)
-\$ActionQueueSaveOnShutdown on   # save messages to disk on shutdown
-\$ActionQueueType LinkedList    # run asynchronously
-\$ActionResumeRetryCount -1    # infinite retries if host is down
+
+        cat >/etc/rsyslog.d/10-loggly.conf <<EOF
+\$WorkDirectory /var/spool/rsyslog  # where to place spool files
+\$ActionQueueFileName loggly        # unique name prefix for spool files
+\$ActionQueueMaxDiskSpace 1g        # 1gb space limit (use as much as possible)
+\$ActionQueueSaveOnShutdown on      # save messages to disk on shutdown
+\$ActionQueueType LinkedList        # run asynchronously
+\$ActionResumeRetryCount -1         # infinite retries if host is down
 
 template(name="LogglyFormat" type="string"
  string="<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [$audius_loggly_token@41058 $audius_loggly_tags \\"$audius_discprov_hostname\\"] %msg%\n")
 
 # Send messages to Loggly over TCP using the template.
 action(type="omfwd" protocol="tcp" target="logs-01.loggly.com" port="514" template="LogglyFormat")
 EOF
+
+        cat >/etc/rsyslog.d/20-file.conf <<EOF
+\$WorkDirectory /var/spool/rsyslog  # where to place spool files
+\$ActionQueueFileName file          # unique name prefix for spool files
+\$ActionQueueMaxDiskSpace 1g        # 1gb space limit (use as much as possible)
+\$ActionQueueSaveOnShutdown on      # save messages to disk on shutdown
+\$ActionQueueType LinkedList        # run asynchronously
+\$ActionResumeRetryCount -1         # infinite retries if host is down
+
+\$outchannel server_log,/var/log/discprov-server.log, 52428800,/audius-discovery-provider/scripts/rotate-log.sh
```

Comment:
do these log files exist outside the container or just inside?

Reply:
No, they exist only inside the container.